modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/shared/sxp_element/index.js

```diff
@@ -210,7 +214,10 @@ function SXPElement({
 					);
 				}
 
-				if (config.name.includes(`${GROUP_ID}s`)) {
+				if (
+					config.name.includes(`${GROUP_ID}s`) ||
+					config.name.includes(`${GROUP_EXTERNAL_REFERENCE_CODE}s`)
```

Comment:
We should create new variables for this instead of adding an `s`.

- `GROUP_IDS`
- `GROUP_EXTERNAL_REFERENCE_CODES`
- `ASSET_CATEGORY_IDS`

An acceptable case would be if this additional of `s` is being used as an identifier of sorts (like some check if the value ends with an `s`). Since this doesn't seem like the case, this could lead to an oversight where we rename the constant but not realizing this `s` needs to be changed too.

Reply:
Sure thing!

Created the separate variables for each of them 👍